flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/FlinkConfigBuilder.java

```diff
@@ -269,6 +270,12 @@ protected FlinkConfigBuilder applyJobOrSessionSpec() throws URISyntaxException {
                 effectiveConfig.set(
                         ApplicationConfiguration.APPLICATION_MAIN_CLASS, jobSpec.getEntryClass());
             }
+
```

Comment:
Can we add a test for this, perhaps in `FlinkConfigBuilderTest`?

Reply:
✅